src/content/get-started/packages-libs.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Organizing Dart code with packages and libraries
+short-title: Packages and libraries
+description: >-
+  Learn how to organize your Dart code into reusable libraries and packages.
+prevpage:
+  url: /get-started/async
+  title: Introduction to async and HTTP
+nextpage:
+  url: /get-started
+  title: Object oriented dart
+---
+
+In this chapter, you'll learn how to refactor your Dart code into
+reusable components by creating a separate package for handling
+command-line arguments. This involves understanding Dart libraries,
+`export` statements, and how to structure your project for better
+organization and maintainability. This refactoring will set the stage
+for building a more advanced command-line application in the next part
+of the guide.
+
+:::secondary What you'll learn
+
+* Create new Dart packages using `dart create -t package`.
+* Structure your project to include local packages.
+* Add local packages as dependencies using the `path` option
+  in `pubspec.yaml`.
+* Use `export` statements to make library declarations available
+  to other packages.
+* Import and use classes from your new package in your `dartpedia`
+  application.
+* Understand the benefits of refactoring code into reusable
+  packages.
+
+:::
+
+## Prerequisites
+
+* Completion of Chapter 3, which covered asynchronous programming
+  and HTTP requests.
+* Basic understanding of code organization principles.
+
+## Tasks
+
+In this chapter, you'll be refactoring the existing `dartpedia` CLI application by
+extracting the command-line argument parsing logic into a separate package
+called `command_runner`. This will improve the structure of your project,
+making it more modular and maintainable.
```

Comment:
It's important that we explicitly explain that there is a "real" package called [`args`](https://pub.dev/packages/args), maintained by the Dart team, that has a class called `CommandRunner` that this tutorial implements for demo purposes, but in a "real project", you should use the `args` package when building a CLI app. 

If this tutorial just used the args package, it would be a _very_ short tutorial.